school/demo/school_demo.xml

```diff
@@ -88,30 +88,30 @@
         <!--Academic Year  Information-->
 
         <record id="demo_academic_year_1" model="academic.year">
-            <field name="name">Academic Year 2011</field>
-            <field name="code">2011</field>
+            <field name="name">Academic Year 2015</field>
+            <field name="code">2015</field>
             <field name="sequence">1</field>
             <field name="grade_id" ref="demo_student_grade_1"/>
-            <field name="date_start">01/01/2011</field>
-            <field name="date_stop">12/31/2011</field>
+            <field eval="str(int(time.strftime('%Y'))-2) +'-01-01'" name="date_start"/>
+            <field eval="str(int(time.strftime('%Y'))-2) +'-12-31'" name="date_stop"/>
         </record>
```

Comment:
Totally wrong....in 2019, this will make year 2015 but start date will be 2017! Think logical. Demo is ok to have 2015 everywhere.